README.md

```diff
@@ -36,10 +36,10 @@ We train a dynamic gap sentence generation model on both C4 and HugeNews, and dy
 | aeslc | 37.69/21.85/36.84 | 37.40/21.22/36.45 | 37.68/21.25/36.51|
 | billsum | 57.20/39.56/45.80 | 57.31/40.19/45.82 | 59.67/41.58/47.59|
 
-The "Mixed & Dynamic" model has the following changes:
+The "Mixed & Stochastic" model has the following changes:
 - trained on both C4 and HugeNews (dataset mixture is weighted by their number of examples). 
 - trained for 1.5M instead of 500k (we observe slower convergence on pretraining perplexity).
-- the model dynamicly choose 15%-45% important sentences to generate
+- the model uniformly sample a gap sentence ratio between 15% and 45%.
 - importance sentences are sampled instead of using a fixed strategy (This is done by adding a 20% noise to importance scores.) 
```

Comment:
maybe remove "using a fixed strategy"? On line 42 "the model uniformly sample" -> "the model uniformly samples".

Reply:
Done.